trait-representation-migration/remove_background_trait_links.py

```diff
@@ -0,0 +1,213 @@
+# Activate Python venv for the script - uncomment to run script on commandline
+# activate_this_file = "/path/to/bin/activate_this.py"
+# execfile(activate_this_file, dict(__file__ = activate_this_file))
+
+import cx_Oracle
+import contextlib
+import argparse
+import sys
+from tqdm import tqdm
+import csv
+import os.path
+
+sys.path.insert(0, '/path/to/gwas_data_sources')
+import gwas_data_sources
+
+import datetime
+
+
+def read_file(filename):
+    '''
+    Read file.
+
+    Args:
+        filename: Name of file provided as a commandline argument.
+
+    Returns:
+        data_map: Dictionary with the STUDY_ID as the key and a 
+        list of EFO_IDs as the dictionary value.
+    '''
+
+    with open(filename, 'r') as file:
+        lines = file.readlines()[1:]
+
+    return lines
+
+
+def process_file_contents(file_data, efo_map, cursor):
+    '''
+    Extract fields of interest from the file and clean-up values.
+
+    Args:
+        file_data: Contents of the file.
+    '''
+
+    count = 0
+
+    for line in file_data:
+        formatted_line = line.split('\t')
+
+        study_accession = formatted_line[2].strip()
```

Comment:
This line assumes this script will only be run on already published studies with accession IDs. If this assumption is correct there's nothing to do.